src/app/dashboard.component/dashboard.component.html

```diff
@@ -0,0 +1,10 @@
+<h3>TODOS</h3>
+<ul class="heroes">
+    <li *ngFor="let todo of todos"
+     [class.hidden]="showT(todo)">
```

Comment:
is there any use of this [class.hidden]="showT(todo)
if not remove this.

Reply:
sir it is used